Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'jekyll'
+gem 'jekyll-feed'
+gem 'jekyll-gist'
+gem 'jekyll-paginate'
+gem 'jekyll-sitemap'
+gem 'jekyll-slim'
+gem 'jemoji'
+
```

Comment:
Extra line

Reply:
Do you like new lines or not? 😅

Reply:
Please, open a PR adding formatters and I will follow them easily